recipe/meta.yaml

```diff
@@ -76,5 +81,6 @@ about:
   dev_url: https://github.com/hep-packaging-coordination/ff
 
 extra:
+  feedstock-name: ff
```

Comment:
For `@conda-forge/core` reviewers: Is setting the `feedstock-name` to what the intended name was viable?

Reply:
If you mess up the feedstock creation, it's set in stone. Too many places in our infrastructure where this appears (all automated) so that manual clean-ups are not possible. The only effect this field has is how the README is rendered for the feedstock.

Reply:
Okay, thanks. Changing to `ff-static` now.